client/my-sites/stats/activity-log-confirm-dialog/index.jsx

```diff
@@ -15,116 +14,57 @@ import Card from 'components/card';
-	static propTypes = {
-		applySiteOffset: PropTypes.func.isRequired,
-		onClose: PropTypes.func.isRequired,
-		onConfirm: PropTypes.func.isRequired,
-		timestamp: PropTypes.number,
-		type: PropTypes.string,
-		icon: PropTypes.string,
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+const ActivityLogConfirmDialog = ( {
+	children,
+	confirmTitle,
+	icon = 'history',
+	onClose,
+	onConfirm,
+	title,
+	translate,
+} ) => (
+	<div className="activity-log-item activity-log-item__restore-confirm">
+		<div className="activity-log-item__type">
+			<ActivityIcon activityIcon={ icon } />
+		</div>
+		<Card className="activity-log-item__card">
+			<h5 className="activity-log-confirm-dialog__title">{ title }</h5>
 
-		// Localize
-		translate: PropTypes.func.isRequired,
-		moment: PropTypes.func.isRequired,
-	};
+			<div className="activity-log-confirm-dialog__highlight">{ children }</div>
 
-	static defaultProps = {
-		type: 'restore',
-		icon: 'history',
-	};
-
-	handleClickCancel = () => this.props.onClose( this.props.type );
-	handleClickConfirm = () => this.props.onConfirm( this.props.type );
-
-	render() {
-		const { applySiteOffset, moment, timestamp, translate, type, icon } = this.props;
-		const activityTime = applySiteOffset( moment.utc( timestamp ) ).format( 'LLL' );
-		const strings = {};
-
-		switch ( type ) {
-			case 'restore':
-				strings.title = translate( 'Rewind Site' );
-				strings.confirm = translate( 'Confirm Rewind' );
-				strings.highlight = translate(
-					'This is the selected point for your site Rewind. ' +
-						'Are you sure you want to rewind your site back to {{b}}%(time)s{{/b}}?',
-					{
-						args: { time: activityTime },
-						components: { b: <b /> },
-					}
-				);
-				break;
-			case 'backup':
-				strings.title = translate( 'Create downloadable backup' );
-				strings.confirm = translate( 'Create download' );
-				strings.highlight = translate(
-					'We will build a downloadable backup of your site at {{b}}%(time)s{{/b}}. ' +
-						'You will get a notification when the backup is ready to download.',
-					{
-						args: { time: activityTime },
-						components: { b: <b /> },
-					}
-				);
-				break;
-		}
-
-		/* eslint-disable wpcalypso/jsx-classname-namespace */
-		return (
-			<div className="activity-log-item activity-log-item__restore-confirm">
-				<div className="activity-log-item__type">
-					<ActivityIcon activityIcon={ icon } />
+			<div className="activity-log-confirm-dialog__button-wrap">
+				<div className="activity-log-confirm-dialog__primary-actions">
+					<Button onClick={ onClose }>{ translate( 'Cancel' ) }</Button>
+					<Button primary onClick={ onConfirm }>
+						{ confirmTitle }
+					</Button>
+				</div>
+				<div className="activity-log-confirm-dialog__secondary-actions">
+					<Button
+						borderless={ true }
+						className="activity-log-confirm-dialog__more-info-link"
+						href="https://help.vaultpress.com/one-click-restore/"
```

Comment:
This needs to be a prop because the link will different for restore, backup, and transfer.

Reply:
Seems like it was already hard-coded for both dialogues. Do we have a different link for the download dialog? If so, I'd be happy to add them in as props here, otherwise maybe in another PR when we actually introduce the second link?

Reply:
No link yet. I spoke today about the docs and they will be taking a new look at AL soon. Since you added a few props now, adding one more, totally related to this, makes sense.

Reply:
prop added in 3e5c68d41